backstory/src/test/java/com/google/sps/story/StoryManagerTest.java

```diff
@@ -85,6 +85,25 @@ public void invalidTemperature() {
     storyManager = new StoryManagerImpl(PREFIX_SAMPLE, SIZE_SAMPLE, 1000.0);
   }
 
+  @Test
+  public void cyclePastBounds() {
+    storyManager = new StoryManagerImpl(PREFIX_SAMPLE, SIZE_SAMPLE, TEMPERATURE_SAMPLE);
+    try{
+      boolean firstCycle = true;
+      
+      //Cycle through all URLs in first cycle.
+      while(firstCycle) {
```

Comment:
Similarly, according to Google style guide there should be a space after while (also one before cycle in the comments).

```suggestion
      while (firstCycle) {
```